src/app/containers/AVPlayer/StyledAudioPlayer.jsx

```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components';
+import {
+  GEL_SPACING,
+  GEL_SPACING_DBL,
+  GEL_SPACING_QUAD,
+} from '@bbc/gel-foundations/spacings';
+import { GEL_GROUP_2_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import AVPlayer from '.';
+
+const StyledAudioPlayer = styled(AVPlayer)`
+  amp-iframe {
```

Comment:
you can just do 
```suggestion
  amp-iframe, div > iframe {
```

and remove the other one if they're the same. also not sure about having this as a separate importable component